scholia/app/templates/pathway.html

```diff
@@ -77,11 +77,13 @@
 
 //Extensible query for muliple, optional values
 const optionalDataValuesSparql =`
-SELECT ?wpid ?organism ?organismLabel
+SELECT ?wpid ?organism ?organismLabel ?description
```

Comment:
Why not ?organismDescription  with automatic description?

Reply:
Ah, we are not getting the organismDescription here. We are getting the description of the item `wd:{{q}} schema:description ?description`.  Is there a better way to retrieve that?